patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -443,6 +443,11 @@
 +
 +
 +    /* ======================================== FORGE START =====================================*/
```

Comment:
Yes, we keep the other version for backwards compatibility. I should mark it deprecated, though.